index.html

```diff
@@ -348,28 +372,38 @@ <h2 id="defining_bidi_keywords">Defining Bidirectional Keywords in Specification
 	"language": "ar"
 }</pre>
 
-<p><strong>Example of a <a>display direction attribute</a>.</strong> If the above JSON were received by a process that was assembling a Web page for display, it might be filling in a template similar to the top line in this example. Here the <code>dir</code> attribute from [[HTML]] is an example of a <a>display direction attribute</a>.</p>
+<p><strong>Example of a <a>display direction attribute</a>.</strong> If the above JSON were received by a process that was assembling a Web page for display, it might be filling in a template similar to the top line in this example to produce markup like the second line. Here the <code>dir</code> attribute from [[HTML]] is an example of a <a>display direction attribute</a>.</p>
```

Comment:
I have a number of comments on section 2.6.

Reply:
'field direction value' refers to the value of the direction field, whereas 'display direction attribute' refers to the attribute, rather than its value.  I think that better terms would be, respectively, 'direction field' and 'direction attribute'.  We could then  talk about the direction field value, etc. where needed.

Reply:
The definitions for the above 2 terms are hard to grasp without examples, but very easy with.  I therefore suggest splitting EXAMPLE 2 so that each of the 2 examples it contains occurs immediately after the appropriate definition.

Reply:
> the title structure has a value direction which represents the field direction of the value field.

???

How about: "... the title structure has a direction field, who's value represents the direction of the title."  Unless you had an RDF-like @ value at the end of the string, i don't think you can say that the direction field represents the content of the value field, though you could say 'which represents the value of the  title's value field" (which is somewhat clunky).

Reply:
> Use the field name...

I'd be inclined to write "Consider using the field name...", since i don't think this is an absolute or strong recommendation like the others - which others we really want people to observe.

Reply:
> indicates a base direction of left-to-right, in exactly the same manner indicated by CSS writing modes [CSS-WRITING-MODES-4]

Remind me why we are pointing to the CSS and HTML specs here, rather than to the UBA?  If they contain something different from the UBA, we should probably explain that.

Reply:
These are all great comments and I must admit that I'm not keen on 'field direction value' and 'display direction attribute'. See what you think of the edits.